evm/v0.6/contracts/dev/VRF.sol

```diff
@@ -0,0 +1,523 @@
+// solhint-disable-next-line compiler-version
```

Comment:
We no longer use the `evm` directory, you should be able to delete it all and just use `evm-contracts` instead.

Reply:
One thing I noticed while investigating this is that `core/eth/a_eth-packr.go` references stale `evm/` paths in my working tree, because I haven't run `make install` since forever. We may want to add a test like `vrf/go_generate_test.go` to ensure that people are getting up-to-date contract wrappers.